app/policies/kondo_policy.rb

```diff
@@ -3,5 +3,25 @@ class Scope < Scope
     def resolve
       scope.all
     end
+
+    def new?
+      current_user.provider?
+    end
```

Comment:
This code is not needed, reason being is that inside the application_policy.rb the `new?` method calls the `create?` method from this file. Would be happy to explain more if its not clear :)